libr/anal/anal.c

```diff
@@ -167,13 +167,13 @@ R_API RAnal *r_anal_new() {
 	anal->opt.depth = 32;
 	anal->opt.noncode = false; // do not analyze data by default
 	r_spaces_init (&anal->meta_spaces, "CS");
-	r_event_hook (anal->meta_spaces.event, R_SPACE_EVENT_UNSET, meta_unset_for);
-	r_event_hook (anal->meta_spaces.event, R_SPACE_EVENT_COUNT, meta_count_for);
+	r_event_hook (anal->meta_spaces.event, R_SPACE_EVENT_UNSET, meta_unset_for, NULL);
```

Comment:
not super fan of adding more parameters

Reply:
you need to pass the right user here though

Reply:
Actually, you don't since you've changed callbacks to use `ev->user`. Sorry for the noise.